### PR TITLE
Consult latest ruleset endpoint

### LINKED ISSUE #33: Consult latest ruleset endpoint
Endpoint: `GET /i/rulesets/<path>`

Expected payload :
```js
{
    "path": "",
    "version": "",
    "signature": {
        "params": [
            {
                "<name>": "<type>"
            }
        ],
        "returnType": "<type>"
    },
    "versions": [
        "",
        ""
    ], // list of string representing all the existing versions of the ruleset
    "rules": [
        {
            "sExpr": "",
            "returnValue": "",
        }
    ]
}
```

## Success
Response: 200

### PR DESCRIPTION
**Pull request by tealeg**
Fix #33 

This PR defines a handler in the UI package to return the latest version of a ruleset, including a list of all its versions.   This handler has also been wired into the `LatestRuleset.vue` ui code, so we can now navigate to rules on the sidebar and have their rules load and display.

### CODE REVIEW

File: ui/app/src/views/LatestRuleset/LatestRuleset.vue [hunk 1]
```diff
@@ -7,7 +7,9 @@
         <v-toolbar color="grey" dark>
           <v-toolbar-title>Parameters</v-toolbar-title>
         </v-toolbar>
-        <v-card class="height-card scroll">
+        <v-card class="height-card scroll"
+                            v-if="typeof(ruleset.signature) != 'undefined'"
```
[the remark this answers is not in the document]
Comment (tealeg) [reply ui/app/src/views/LatestRuleset/LatestRuleset.vue]: No, it didn't, and I don't know why.

File: ui/handler.go [hunk 2]
```diff
@@ -120,6 +121,60 @@ func (h *internalHandler) handleNewRulesetRequest(w http.ResponseWriter, r *http
 
 }
 
+func (h *internalHandler) handleSingleRuleset(w http.ResponseWriter, r *http.Request) {
+	logger := reghttp.LoggerFromRequest(r)
+	path := strings.TrimPrefix(r.URL.Path, "/rulesets/")
+	logger.Debug().Msg(fmt.Sprintf("PATH == %q", path))
```
Comment (YassineE) [inline ui/handler.go]: I think this one is forgotten?

Comment (YassineE) [inline ui/handler.go]: I think this one is forgotten?

Comment (tealeg) [reply ui/handler.go]: Done

**Pull request merged**